src/domain/post/toggleRating.ts

```diff
@@ -10,7 +10,7 @@ export default async function toggleRating(requester: Requester, postId: string)
 
     try
     {
-        rating = await updateRating(requester, postId, undefined);
```

Comment:
I think we should restore this, and make the parameter not optional anymore to make it more explicit.